webapp/src/components/Metrics/PerformanceAnalysis.tsx

```diff
@@ -23,12 +24,14 @@ import SeeMoreLess, {
   useMoreLess,
 } from "components/SeeMoreLess";
 import { Table, Column } from "components/Table";
+import VisualBar from "components/VisualBar";
 import React from "react";
 import { getMetricsPerFilterEndpoint } from "services/api";
-import { DatasetSplitName, MetricsPerFilterValue } from "types/api";
+import { DatasetSplitName, MetricsPerFilterValue, Outcome } from "types/api";
```

Comment:
That new import is not used, resulting in a warning.
```suggestion
import { DatasetSplitName, MetricsPerFilterValue } from "types/api";
```